src/textacy/constants.py

```diff
@@ -196,3 +241,9 @@
 )
 
 RE_ALNUM: Pattern = re.compile(r"[^\W_]+")
+
+# regexes for quote detection prep
+ALL_QUOTES = '‹「`»」‘"„›”‚’\'』『«“'
+DOUBLE_QUOTES = '‹「」»"„『”‚』›«“'
+ANY_DOUBLE_QUOTE_REGEX = r"[{}]".format(DOUBLE_QUOTES)
+DOUBLE_QUOTES_NOSPACE_REGEX = r"(?<=\S)([{}])(?=\S)".format(DOUBLE_QUOTES)
```

Comment:
elsewhere in this module, i've actually compiled the constant regex patterns. could that be done for these two, as well? e.g. `RE_ANY_DOUBLE_QUOTE = re.compile(rf"{DOUBLE_QUOTES}")`

note: i was having trouble getting the that example compiled regex pattern to work -- assuming user error on my end